php/flights.php

```diff
@@ -197,15 +204,15 @@
     }
 
     if ($export == "export" || $export == "backup") {
-        $note = "\"" . $note . "\"";
+        $note = "\"$note\"";
```

Comment:
We can stop doing this manually if we swapped to some proper csv functions in php...